projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.ts

```diff
@@ -374,6 +389,25 @@ export class IgxPivotGridComponent extends IgxGridBaseDirective implements OnIni
         }
     }
 
+    /**
+     * Gets/Sets the values clone strategy of the pivot grid when assigning them to different dimensions.
+     *
+     * @example
+     * ```html
+     *  <igx-pivot-grid #grid [data]="localData" [pivotValueCloneStrategy]="customCloneStrategy"></igx-pivot-grid>
+     * ```
+     */
+    @Input()
+    public get pivotValueCloneStrategy(): IDataCloneStrategy {
```

Comment:
Oh, okay. 
So this becomes an actual input. In that case I suppose disregard part of my previous comment since I assume the idea _is_ to make the strategy public. In which case.. do document it a bit at least to explain what the use of it is.